tests/lib/outputs/slack_test.py

```diff
@@ -65,7 +65,7 @@ def test_create_message_blocks(self):
                 "type": "section",
                 "text": {
                     "type": "mrkdwn",
-                    "text": f"Hey there 👋 \n I'm *Prowler*, _the handy cloud security tool_ :cloud::key:\n\n I have just finished the security assessment on your {aws_identity} with a total of *{stats['findings_count']}* findings.",
+                    "text": f"Hey there 👋 \n I'm *Prowler*, _the handy multi-cloud security tool_ :cloud::key:\n\n I have just finished the security assessment on your {aws_identity} with a total of *{stats['findings_count']}* findings.",
```

Comment:
Could you please move this text to a constant since it is always the same? Save it in the `prowler/lib/outputs/slack.py` and then use it here.

Reply:
Done